apps/token-manager/app/src/components/InfoBoxes.js

```diff
@@ -110,13 +111,17 @@ function InfoBoxes({
           items={stakes}
           renderLegendItem={({ item: account }) => {
             const isCurrentUser = addressesEqual(account, connectedAccount)
+            const [label, showLocalIdentityModal] = useIdentity(account)
+            
             return (
               <div>
                 <LocalIdentityBadge
                   entity={account}
                   connectedAccount={isCurrentUser}
+                  customLabel={!Boolean(label) && isCurrentUser && 'YOU'}
+                  labelStyle={(!Boolean(label) && isCurrentUser ? `color: ${theme.tagIndicatorContent};` : '')}
                 />
-                {isCurrentUser && <You />}
+                {isCurrentUser && Boolean(label) && <You />}
```

Comment:
Ah, I wonder if we still need to show this? I understood the change as always showing just the identity badge.

Reply:
So, when we have a custom label (like in the first case of the picture) I have to show the custom label or 'YOU'? I think the two things are relevant
I am only replacing for 'YOU' in the badge when it doesnt have a custom label

Reply:
Gotcha. I think keeping the "you" makes sense then. Let's double check what happens if you use a long label; ideally we will keep the "you" full width and shrink the label's size.